src/app/footer/footer.component.scss

```diff
@@ -0,0 +1,215 @@
+@import '../../variables' ;
+
+.footer {
+    flex-shrink: 0;
+    display: block;
+}
+
+.pt-5 {
+    padding-top: 3rem !important;
+}  
+
+.col-md-4 {
+    flex: 0 0 auto;
+    width: 33.33333333%;
```

Comment:
if we use md-4, by default bootstrap will divide the width into 3 parts with 33.33% for each. Why do we need to add it again?

Reply:
@akashsinha13, yes , removed the code.